docs/modules/plugins/partials/plugin-web-app-locators.adoc

```diff
@@ -20,6 +20,12 @@
 |https://developer.mozilla.org/en-US/docs/Web/XPath[Xpath to locate an element]
 |xpath(//a)
 
+|`exactXPath`
+|https://developer.mozilla.org/en-US/docs/Web/XPath[Xpath to locate an element].
+Unlike `xPath` type doesn't add https://developer.mozilla.org/en-US/docs/Web/XPath/Functions/normalize-space[space normalization] and
+leaving the handling of spaces up to user
+|exactXPath(//li[text()='ID:&nbsp;&nbsp;&nbsp;&nbsp;testId'])
```

Comment:
is it correct xpath? I'm interested particularly in `&nbsp;`

Reply:
> is it correct xpath? I'm interested particularly in `&nbsp;`

For end users - yes (but in the page source code will be still `'ID:&nbsp;&nbsp;&nbsp;&nbsp;testId'`).
![image](https://github.com/vividus-framework/vividus/assets/58571912/a31d4f89-4170-4fbe-a330-d1ccb69b5e90)

AsciiDoc using auto-normalization for double spaces (except for code snippet). I didn't find how to prevent this kind of formatting inline in the table.
By the way, it can be replaced with built-in attribute for non-breaking space - "{nbsp}".

Reply:
got it, thanks, any approach works (both `&nbsp;` and `{nbsp}`), it's up to you which one to follow